src/components/court-order-poa/CourtOrderPoa.vue

```diff
@@ -87,24 +84,21 @@ export default class CourtOrderPoa extends Vue {
     return this.$refs.courtNumRef.validate()
   }
 
-  @Watch('autoValidation')
   @Watch('planOfArrangement')
   @Watch('courtOrderNumber')
   @Watch('courtOrderNumberRequired')
   validateCourtNum (): void {
-    if (this.autoValidation) {
-      // Apply TextField rules
-      this.courtOrderNumRules = [
-        (v: string) => (!v || !/^\s/g.test(v)) || 'Invalid spaces', // leading spaces
-        (v: string) => (!v || !/\s$/g.test(v)) || 'Invalid spaces', // trailing spaces
-        (v: string) => (!v || !(v.length < 5)) || 'Court order number is invalid',
-        (v: string) => (!v || !(v.length > 20)) || 'Court order number is invalid'
-      ]
-      if (this.courtOrderNumberRequired || this.planOfArrangement) {
-        this.courtOrderNumRules.push((v: string) => !!v || 'A Court Order number is required')
-      }
-      this.$refs.courtNumRef.validate()
-    } else this.clearValidations()
+    // Apply TextField rules
+    this.courtOrderNumRules = [
+      (v: string) => (!v || !/^\s/g.test(v)) || 'Invalid spaces', // leading spaces
+      (v: string) => (!v || !/\s$/g.test(v)) || 'Invalid spaces', // trailing spaces
+      (v: string) => (!v || !(v.length < 5)) || 'Court order number is invalid',
+      (v: string) => (!v || !(v.length > 20)) || 'Court order number is invalid'
+    ]
+    if (this.courtOrderNumberRequired || this.planOfArrangement) {
+      this.courtOrderNumRules.push((v: string) => !!v || 'A Court Order number is required')
+    }
```

Comment:
Could these rules be static (readonly const properties)?

Reply:
Oh wait ... that last rule is conditional, so they can't be static.

But they could be a getter.

Reply:
As we all know, validation is tricky and fragile, so if this code works then I'm OK to leave it as-is.